Pi_Code/Controller_Operations.py

```diff
@@ -0,0 +1,113 @@
+from multiprocessing import Process
```

Comment:
Remove all images, this is not the place for them

Reply:
Images removed.